From: dusty@dustymabe.com
Subject: dusty-grub2-btrfs-default-to-follow-default.patch

Add in dusty-grub2-btrfs-default-to-follow-default.patch

diff -up grub-2.02~beta2/grub-core/fs/btrfs.c.orig grub-2.02~beta2/grub-core/fs/btrfs.c
--- grub-2.02~beta2/grub-core/fs/btrfs.c.orig	2015-07-13 00:53:40.782000000 +0000
+++ grub-2.02~beta2/grub-core/fs/btrfs.c	2015-07-13 00:54:00.301000000 +0000
@@ -913,7 +913,7 @@ grub_btrfs_mount (grub_device_t dev)
 {
   struct grub_btrfs_data *data;
   grub_err_t err;
-  const char *relpath = grub_env_get ("btrfs_relative_path");
+  const char *relpath = "y";
 
   if (!dev->disk)
     {
@@ -1417,7 +1417,7 @@ find_path (struct grub_btrfs_data *data,
   char *path_alloc = NULL;
   char *origpath = NULL;
   unsigned symlinks_max = 32;
-  const char *relpath = grub_env_get ("btrfs_relative_path");
+  const char *relpath = "y";
 
   follow_default = 0;
   origpath = grub_strdup (path);
